src/OpenTK.Graphics/OpenTK.Graphics.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
This seems like quite a big change to be making. I assume this is for the SSE stuff? Might be better to make it optional behind an `#if` if targeting Core 3.1 instead.